docs/usage.rst

```diff
@@ -279,6 +280,128 @@ exception thrown by the backend.
         display(str(ex.exception))
 
 
+.. _backend-interface:
+
+Backend interface
+-----------------
```

Comment:
This is a nice example for an interface.

At the beginning of the usage section we state at the moment

![image](https://github.com/audeering/audbackend/assets/173624/0a1dcfb6-3e23-4886-8101-22105885241a)

Maybe, you can update that and mention also the third (interface) sub-section there.

Reply:
Changed to:

![image](https://github.com/audeering/audbackend/assets/10383417/54eace53-b242-471a-8659-581e46804196)